packages/config/config/devices/0x0346/alarm_keypad_gen2.json

```diff
@@ -1,16 +1,19 @@
-// Ring KEYPAD V2
-// Ring Keypad v2
+// Ring - / 4AK-1SZ-OEU0
+// Alarm Keypad (2nd generation)
 {
 	"manufacturer": "Ring",
 	"manufacturerId": "0x0346",
-	"label": "KEYPAD V2",
-	"description": "Ring Keypad v2",
+	"label": "- / 4AK-1SZ-OEU0",
+	"description": "Alarm Keypad (2nd generation)",
 	"devices": [
 		{
 			"productType": "0x0101",
-			"productId": "0x0301",
-			"zwaveAllianceId": 3870
```

Comment:
We did import this from the ZWA website, so please keep the ID.

Reply:
Added on both productIds

Reply:
ZWA only has the product ID 0x0301, so we should only add the reference to that one